bin/tachyon

```diff
@@ -282,6 +283,12 @@ elif [[ ${COMMAND} == "thriftGen" ]]; then
     thrift --gen java -out ${bin}/../common/src/main/java/. ${bin}/../common/src/thrift/${src_file}
   done
   exit 0
+elif [[ ${COMMAND} == "protoGen" ]]; then
```

Comment:
hmm, this entire file seems to be violating that style and it's worse to be inconsistent. Adding quotes everywhere might break things, so I'd rather not make that change in this PR. Can we delegate the task of quoting variable expansions to a separate PR?